xml/System.Windows.Media/SkewTransform.xml

```diff
@@ -24,7 +24,7 @@
 ## Remarks  
  A <xref:System.Windows.Media.SkewTransform> is useful for creating the illusion of 3-dimensional depth in a [!INCLUDE[TLA#tla_2d](~/includes/tlasharptla-2d-md.md)] object.  
   
- **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.SkewTransform> class provides several special features: <xref:System.Windows.Media.SkewTransform> objects can be declared as , shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
+ **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.SkewTransform> class provides several special features: <xref:System.Windows.Media.SkewTransform> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md),shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```

Comment:
```suggestion
 **Freezable Features:** Because it inherits from the <xref:System.Windows.Freezable> class, the <xref:System.Windows.Media.SkewTransform> class provides several special features: <xref:System.Windows.Media.SkewTransform> objects can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md), shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```